src/theme.js

```diff
@@ -1,3 +1,13 @@
 const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiTheme');
+const { default: amber } = require('@material-ui/core/colors/amber');
```

Comment:
Just our (via hapi) convention to `require()` things in pascal-case:

```suggestion
const { default: Amber } = require('@material-ui/core/colors/amber');
```